test/integration/work/statusfeedback_test.go

```diff
@@ -242,8 +242,8 @@ var _ = ginkgo.Describe("ManifestWork Status Feedback", func() {
 					ResourceIdentifier: workapiv1.ResourceIdentifier{
 						Group:     "apps",
 						Resource:  "deployments",
-						Namespace: commOptions.SpokeClusterName,
-						Name:      "deploy1",
+						Namespace: "*",
+						Name:      "*1",
```

Comment:
we should have some more tests for wildcard case.

Reply:
added

Reply:
We should also have another test with multiple matched options.

Reply:
done